src/popup/MainContainer.tsx

```diff
@@ -57,41 +65,65 @@ export default class MainContainer extends Component<IProps, {}> {
             <Route exact path="/add-token" component={AddToken} />
           </Switch>
         </Router>
+        <UnexpectedErrorDialog store={ this.props.store } />
```

Comment:
you can inject a store into a stateless component:
https://stackoverflow.com/a/42947477/2917742